Form/Validation/TextValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DUCK.Form.Fields;
+using UnityEngine;
+
+namespace DUCK.Form.Validation
+{
+	[RequireComponent(typeof(TextField))]
+	public abstract class TextValidator : AbstractValidator
+	{
+		protected TextField textField;
```

Comment:
`{ get; private set; }`

You probably don't want your child classes to reassign this field.